mcs_benchmark_data/models/benchmark_bootstrap.py

```diff
@@ -0,0 +1,21 @@
+from typing import NamedTuple, Tuple, Optional
+
+from rdflib import Graph
+from rdflib.resource import Resource
+from mcs_benchmark_data.namespace import MCS, SCHEMA
+
+from mcs_benchmark_data._model import _Model
+from mcs_benchmark_data.models.benchmark_dataset import BenchmarkDataset
+from mcs_benchmark_data.models.submission import Submission
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass(frozen = True)
+class BenchmarkBootstrap(_Model):
+    '''A collection of datasets composing a benchmark'''
+    name: str
+    abstract: str
+    authors: Tuple[str, ...]
+    datasets: Tuple[BenchmarkDataset, ...]

```

Comment:
I don't think you really need datasets or submissions.

One thing you could do here is use the file system to give you metadata.

For example,

```
data/extracted/benchmark/commonsense_qa/metadata.json (about the benchmark)

data/extracted/benchmark/commonsense_qa/dataset/<dataset id>/metadata.json (about the dataset, if it can't be synthesized as I mentioned in another comment)

data/extracted/benchmark/commonsense_qa/dataset/<dataset id>/submission/<submission id>/metadata.json
```

Probably not exact, but you get the picture.